examples/README.md

```diff
@@ -2,19 +2,20 @@
 
 The following examples are added for your convenience. Each of these files
 would be added to a .github/workflows folder to be run on some [github event](https://help.github.com/en/actions/reference/events-that-trigger-workflows).
+Note that the files reference the "master" branch of the repository for consistency, however you should change this to an actual [release](https://github.com/urlstechie/urlchecker-action/releases) for your deployment.
 
 ## General
 
 - [GitHub Checkout](urlchecker-checkout.yml): the most likely case of usage for this action is checking out the repository that the action is running for, meaning we use the active branch for the check.
 - [GitHub Clone](urlchecker-clone.yml): while it's more a niche use case, you might want to clone one or more repos to check for an action run.
 
-## White Listing
+## Excluding Files and Folders
 
-- [urlchecker-whitelist-files.yml](urlchecker-whitelist-files.yml): in this example, we have a repository where we want to check only a README.md file at the root, and importantly, skip over an entire subfolder that serves a rendered site at docs. We want to run the check whenever someone opens a pull request.
+- [urlchecker-exclude-files.yml](urlchecker-exclude-files.yml): in this example, we have a repository where we want to check only a README.md file at the root, and importantly, skip over an entire subfolder that serves a rendered site at docs. We want to run the check whenever someone opens a pull request.
 
 ## Include Files
 
-- [urlchecker-include-files.yml](urlchecker-include-files.yml): as an alternative to white listing files or patterns, you can specify an explicit file path or pattern to check. This is useful if you want to set some comma separated listing of files or patterns in another step, and then set for the action here.
+- [urlchecker-include-files.yml](urlchecker-include-files.yml): as an alternative to excluding files or patterns, you can specify an explicit file path or pattern to check. This is useful if you want to set some comma separated listing of files or patterns in another step, and then set for the action here.
```

Comment:
*to exclude instead of excluding.

Reply:
It's referencing the action (verb) of excluding files, not the field name (otherwise it would say explicitly `exclude_files`, so this is correct I think. But I do see it could be said either way!

Reply:
you are the native speaker here so we will go with your choice ;)

Reply:
Haha, sounds good :)